### PR TITLE
Fixed: HttpHelper.UrlEncode didn't encode long strings.

### PR DESCRIPTION
**Pull request by PSanetra**
Fixed: HttpHelper.UrlEncode didn't encode strings, which were longer than 1000 characters.

### CODE REVIEW

File: Source/Facebook/HttpHelper.cs [hunk 1]
```diff
@@ -1318,7 +1318,7 @@ public static string UrlEncode(string s)
                     ? Uri.EscapeDataString(s.Substring(MAX_LIMIT*i, MAX_LIMIT))
                     : Uri.EscapeDataString(s.Substring(MAX_LIMIT*i)));
             }
-            return s;
```
Comment (prabirshrestha) [inline Source/Facebook/HttpHelper.cs]: good catch

File: Source/Facebook.Tests/HttpHelperTests.cs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Text;
+using System.Threading;
```
Comment (prabirshrestha) [inline Source/Facebook.Tests/HttpHelperTests.cs]: why is this included?

**Comment by prabirshrestha** [comment]
@PSanetra can you also send a PR to https://github.com/facebook-csharp-sdk/http-helper/blob/master/HttpHelper.cs.pp

File: Source/Facebook.Tests/HttpHelperTests.cs (hunk 2 above)
Comment (PSanetra) [inline Source/Facebook.Tests/HttpHelperTests.cs]: Sorry, I might have tested something with async tasks and forgot to delete this line again.

**Comment by PSanetra** [comment]
I created the PR https://github.com/facebook-csharp-sdk/http-helper/pull/22 for the same fix.

**Comment by prabirshrestha** [comment]
@PSanetra thanks for the PR. A new version has been pushed and available for download at https://www.nuget.org/packages/Facebook/7.0.5-beta
